src/config/menu.js

```diff
@@ -18,7 +18,7 @@ const items = [
   'blocks',
   'transactions',
   'addresses',
-  'tokens',
+  'tok3ns',
```

Comment:
is this a typo?

Reply:
No, the commit introducing it makes its usage clear: it's to test the workflow. https://github.com/rsksmart/rsk-explorer/pull/189/commits/c1052cb02e98e34bf78e3f529c44474a57c65ef0 .